.github/workflows/release.yml

```diff
@@ -1,59 +1,179 @@
-    runs-on: ubuntu-latest
-    needs: [ build ]
-    steps:
-      - uses: actions/download-artifact@v2
-        name: Download artifacts
+      - name: Get project dependencies
+        run: go mod download
+ 
+      - name: Build Transfersh
+        run: |
+          mkdir -p build_assets
+          go build -tags netgo -ldflags "-X github.com/dutchcoders/transfer.sh/cmd.Version=${GITHUB_REF##*/} -a -s -w -extldflags '-static'" -o build_assets/transfersh-${GITHUB_REF##*/}-${ASSET_NAME}
+    
+      - name: Build Mips softfloat Transfersh
+        if: matrix.goarch == 'mips' || matrix.goarch == 'mipsle'
+        run: |
+          GOMIPS=softfloat go build -tags netgo -ldflags "-X github.com/dutchcoders/transfer.sh/cmd.Version=${GITHUB_REF##*/} -a -s -w -extldflags '-static'" -o build_assets/transfersh-softfloat-${GITHUB_REF##*/}-${ASSET_NAME}
+
+      - name: Rename Windows Transfersh
+        if: matrix.goos == 'windows'
+        run: |
+          cd ./build_assets || exit 1
+          mv transfersh-${GITHUB_REF##*/}-${ASSET_NAME} transfersh-${GITHUB_REF##*/}-${ASSET_NAME}.exe
+
+      - name: Prepare to release
+        run: |
+          cp ${GITHUB_WORKSPACE}/README.md ./build_assets/README.md
+          cp ${GITHUB_WORKSPACE}/LICENSE ./build_assets/LICENSE
+
+      - name: Create Gzip archive
+        shell: bash
+        run: |
+          pushd build_assets || exit 1
+          touch -mt $(date +%Y01010000) *
+          tar zcvf transfersh-${GITHUB_REF##*/}-${ASSET_NAME}.tar.gz *
+          mv transfersh-${GITHUB_REF##*/}-${ASSET_NAME}.tar.gz ../
+          FILE=`find . -name "transfersh-${GITHUB_REF##*/}-${ASSET_NAME}*"`
+          DGST=$FILE.sha256sum
+          echo `sha256sum $FILE` > $DGST
+          popd || exit 1
+          FILE=./transfersh-${GITHUB_REF##*/}-${ASSET_NAME}.tar.gz
+          DGST=$FILE.sha256sum
+          echo `sha256sum $FILE` > $DGST
+
+      - name: Change the name
+        run: |
+          mv build_assets transfersh-${GITHUB_REF##*/}-${ASSET_NAME}
+
+      - name: Upload files to Artifacts
+        uses: actions/upload-artifact@v2
         with:
-          name: artifacts
-          path: ./artifacts
-      - name: Publish artifacts
+          name: transfersh-${{ steps.get_filename.outputs.GIT_TAG }}-${{ steps.get_filename.outputs.ASSET_NAME }}
+          path: |
+            ./transfersh-${{ steps.get_filename.outputs.GIT_TAG }}-${{ steps.get_filename.outputs.ASSET_NAME }}/*
+
+      - name: Upload binaries to release
         uses: softprops/action-gh-release@v1
+        if: github.event_name == 'release'
         with:
-          files: './artifacts/*'
+          files: |
+            ./transfersh-${{ steps.get_filename.outputs.GIT_TAG }}-${{ steps.get_filename.outputs.ASSET_NAME }}.tar.gz*
+            ./transfersh-${{ steps.get_filename.outputs.GIT_TAG }}-${{ steps.get_filename.outputs.ASSET_NAME }}/./transfersh-${{ steps.get_filename.outputs.GIT_TAG }}-${{ steps.get_filename.outputs.ASSET_NAME }}*
```

Comment:
are you sure about this path?

Reply:
I don't know why I hit one `.`, maybe it was too sleepy at the time. But I didn't notice it because it worked well...
https://github.com/wc7086/transfer.sh/releases/tag/v1.3.3
already edited.